cli/enginecli/flows.py

```diff
@@ -0,0 +1,152 @@
+import argparse
+from enum import Enum
+import json
+import os
+import sys
+import time
+from typing import Any, Dict, Optional, List, Set
+
+from enginecli.core import lootbox_item_to_tuple
+
+from .MockErc20 import MockErc20
+from brownie import network
+
+from . import (
+    abi,
+    Lootbox,
+    ITerminus,
+)
+
+MAX_UINT = 2**256 - 1
+
+ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
+
+
+def handle_create_equippable_pools_from_config(args: argparse.Namespace) -> None:
+    with open(args.config_file, "r") as f:
+        config = json.load(f)
+
+    if type(config) is not list:
+        raise Exception("Config must be a list of equippable items.")
+
+    network.connect(args.network)
+    transaction_config = ITerminus.get_transaction_config(args)
+
+    equipment = ITerminus.ITerminus(args.equipment_address)
+    print("Started with " + str(equipment.total_pools()) + " pools.")
+
+    print("{Item Id} => {Pool Id}")
+    for item in config:
+        equipment.create_pool_v2(
+            MAX_UINT, True, True, item["metadata"], transaction_config
+        )
+        pool_id = equipment.total_pools()
+        print(item["id"] + " => " + str(pool_id))
+
+    print("Ended with " + str(equipment.total_pools()) + " pools.")
+
+
+def handle_create_equippables_lootbox_from_config(args: argparse.Namespace) -> None:
```

Comment:
Maybe it doesn't matter that they are `equippables`. I would do maybe `handle_create_items_lootbox_from_config`. The `items` denotes that they are Terminus tokens.

Reply:
Renamed clis.